src/icos_config/forever.js

```diff
@@ -0,0 +1,77 @@
+import { toPromise, formatNumber } from '../utils';
+import { convertWeb3Value } from '../utils/web3';
+
+export default {
+  crowdSaleTokenContract: '0xFB072e58741738fC26E86D2c49db3179Ea8d5D55',
+  tokenContract: '0xFDb1D263112b0A668dCe5748d961B4851Cfca8Ee',
```

Comment:
can you change it to small case as well?
`0xfdb1d263112b0a668dce5748d961b4851cfca8ee`